features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/listing/template.jag

```diff
@@ -88,7 +88,7 @@
                     path = jagg.getMappedUrl("/site/pages/item-info.jag") + "?name=" + api.name + "&version=" + api.version + "&provider=" + api.provider+urlPostfix;
 
                     var nameSubstring = api.name;
-                    if (api.name.length > 15) {
```

Comment:
Make it a practice to define constants for these type of values, "15" should be one of them here.